examples/blank/README.md

```diff
@@ -1,10 +1,10 @@
-# React-Static - Basic Example
+# React-Static - Blank Example
 
-This example is the most basic version of react-static available. It includes:
+This example is the blank version of react-static available. It includes:
```

Comment:
Great find!
Maybe this could be changed even further, e.g. _"This example is a blank version of react-static. It includes:"_?